gateway-service-definitions/src/main/resources/services/yarnui/2.7.0/rewrite.xml

```diff
@@ -207,7 +207,7 @@
 </filter>
 <filter name="YARNUI/yarn/outbound/filter/cluster">
     <content type="*/html">
-        <apply path="(https?://[^/':,]+:[\d]+)?/ws/v1/cluster/apps/application" rule="YARNUI/yarn/outbound/apps/cluster1"/>
+        <apply path="https?://[^/':,]+:[\d]+/ws/v1/cluster/apps/application" rule="YARNUI/yarn/outbound/apps/cluster1"/>
```

Comment:
My regex skills are rusty :( can you explain briefly what changed here.

Reply:
So, the following two regexes:
```
(https?://[^/':,]+:[\d]+)?/ws/v1/cluster/apps/application
/ws/v1/.*
```
Both match the following link of the kill button in YARN.
```
/ws/v1/cluster/apps/application_1611223368889_0003/state
```
You can confirm this here https://regex101.com/r/QkGNeb/1

Because of this, both the `outbound/apps/cluster1` and `outbound/ws1` rules got applied on the link, resulting in a broken URL.

What this change does is make `https?://[^/':,]+:[\d]+` an _explicit_ match, rather than an optional one, so the `outbound/apps/cluster1` rule will now only ever be applied to links in the form of:
```
http://some-internal-domain:1234/ws/v1/cluster/apps/application
```

Reply:
@moresandeep @iain-buclaw-sociomantic 's explanation seems reasonable to me. Shall we merge this PR?